pkg/scd/repos/repos.go

```diff
@@ -53,6 +53,12 @@ type Subscription interface {
 	IncrementNotificationIndices(ctx context.Context, subscriptionIds []dssmodels.ID) ([]int, error)
 }
 
+type UssAvailability interface {
+	GetUssAvailability(ctx context.Context, id string) (*scdmodels.UssAvailabilityStatus, error)
```

Comment:
Along with the comment above, I think typing `id` as a `dssmodels.Manager` will provide a clearer hint to developers in terms of usage.